src/app/app.component.spec.ts

```diff
@@ -1,31 +1,16 @@
-import { TestBed, async } from '@angular/core/testing';
-import { AppComponent } from './app.component';
+import { TestBed, async } from "@angular/core/testing";
```

Comment:
this and others should be single quoted:

```suggestion
import { TestBed, async } from '@angular/core/testing';
```